docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -436,7 +436,22 @@ The connector supports pushdown for a number of operations:
 
 .. include:: join-pushdown-enabled-true.fragment
 
-.. include:: no-pushdown-text-type.fragment
+Predicate pushdown support
+""""""""""""""""""""""""""
+
+The connector supports pushdown of predicates on ``VARCHAR`` and ``NVARCHAR``
+columns if the underlying columns in SQL Server use a case-sensitive `collation
+<https://learn.microsoft.com/en-us/sql/relational-databases/collations/collation-and-unicode-support?view=sql-server-ver16>`_.
+
+The following operators are pushed down:
+
+- ``=``
+- ``<>``
+- ``IN``
+- ``NOT IN``
+
+Operators are not pushed down and correct results are not preserved for columns
+using a case-insensitive collation.
```

Comment:
```suggestion
Operators are not pushed down for columns using a case-insensitive collation
to preserve correct results.
```

the current sentence reads the opposite (i.e. results are incorrect)

Reply:
It's the "to preserve correct results" that needs to be changed to sound less awkward. Since I'm having trouble improving the flow of the sentence without changing the meaning, please help me understand this process better.

For columns using a case-insensitive collation, operators are not pushed down, which preserves correct results. Is that right or wrong?

Reply:
> For columns using a case-insensitive collation, operators are not pushed down, which preserves correct results.

Yes, this is the behaviour and intention.